engine/src/flutter/shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -781,6 +781,18 @@ - (void)accessibilityElementDidLoseFocus {
   }
 }
 
+- (BOOL)accessibilityRespondsToUserInteraction {
+  if ((self.node.actions & ~flutter::kSystemActions) != 0) {
```

Comment:
a bit hard to read here. mind adding a comment?

Reply:
if im reading it right, if a node has actions (other than just system actions), then it should respond to user interaction?

Reply:
added a comment!